### DISCUSSION TITLE
Release a new version of Danger

### DISCUSSION THREAD

**Discussion by mathroule** [danger/danger#1467] [General]
Hi

How to get a new release of Danger Gem published?

I need the recent changes for supporting Octokit 7.

Thanks

**Comment by orta** [comment]
Only I really make gem releases ATM, open to more long-term contributors who have been doing it for a while having the capabilities to push to rubygems

**Reply by mathroule** [reply]
I have permission to create GitHub releases and push bump commit. But I cannot push to RubyGems.

**Reply by orta** [reply]
Yep, given that neither of those generally would be considered making the actual release I've been OK with that - Danger was built on the [moya contributor system](https://github.com/Moya/contributors/blob/0d5e80682b2377bdca72585eda9ce83467bee3c4/README.md) where folks can get write access pretty easily but we gate giving deployment access until folks have been effectively maintaining the project and gained long-term trust

**Comment by mathroule** [comment]
Thanks @orta for releasing version 9.4.0
